tests/test_integrated_channels/test_integrated_channel/test_exporters/test_learner_data.py

```diff
@@ -675,7 +673,7 @@ def test_learner_exporter_with_skip_transmitted(
         # Check that LMS enrollment populated as part of model used in audit check:
         expected_result = mock_course_enrollment_class.objects.get.return_value
         self.assertEqual(expected_result, enterprise_course_enrollment.course_enrollment)
-        assert mock_course_enrollment_class.objects.get.call_count == 2
```

Comment:
I am not 100% sure why this mock call count went from 2 to 1. but I did debug through the code while running the test and verified that the queryset is correct and the rest of the stuff works as expected. Ideas?